tracer/build/_build/Build.VariableGenerations.cs

```diff
@@ -49,7 +49,13 @@ void GenerateConditionVariableBasedOnGitChange(string variableName, string[] fil
                     const string baseBranch = "origin/master";
                     bool isChanged;
                     var forceExplorationTestsWithVariableName = $"force_exploration_tests_with_{variableName}";
-                    if (bool.Parse(Environment.GetEnvironmentVariable(forceExplorationTestsWithVariableName) ?? "false"))
+
+                    if (Environment.GetEnvironmentVariable("Build.Reason") == "Schedule" && bool.Parse(Environment.GetEnvironmentVariable("isMainBranch") ?? "false"))
```

Comment:
LGTM. Thinking out loud: I wonder if we should remove the `Build.Reason` requirement too and _always_ build these on _every_ master? 🤔 This is probably sufficient for now though

Reply:
This means, that every completed PR will run those tests 🤔 
Isn't it too much?

Reply:
>This means, that every completed PR will run those tests 🤔

Yep

> Isn't it too much?

🤷‍♂️ 😄

Reply:
> This means, that every completed PR will run those tests 

Normally, those tests should not fail if we haven't changed the code. So it would only increase flakiness and the stress to believe we may have break something on each merge, whereas the actual chances of breaking are super super low, no?

Reply:
>  chances of breaking are super super low
They are low, I agree.
However, we have already seen this happened, for example I remember the `exploration tests` being broken for a while.

Anyway, I think running those tests on a daily basis is good enough for now, and if we want to increase the level of confidence, we can do it later.